.travis.yml

```diff
@@ -10,19 +10,24 @@ php:
   - 7.2
   - 7.3
   - 7.4
-  - hhvm
+  - hhvm-3.18
   - nightly
 
 sudo: false
 
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 matrix:
   allow_failures:
-    - php: hhvm
     - php: nightly
+    - php: hhvm
```

Comment:
This doesn't actually do anything since it doesn't match the name `hhvm-3.18`. In any case, both this and `7.4` should be removed from `allow_failures`.